src/Features/VisualBasic/Portable/Completion/CompletionProviders/NamedParameterCompletionProvider.vb

```diff
@@ -85,13 +85,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
                 Dim unspecifiedParameters = parameterLists.SelectMany(Function(pl) pl).
                                                            Where(Function(p) Not existingNamedParameters.Contains(p.Name))
 
-                Dim text = Await document.GetTextAsync(cancellationToken).ConfigureAwait(False)
+                Dim rightToken = semanticModel.SyntaxTree.FindTokenOnRightOfPosition(position, cancellationToken)
+                Dim shouldAddColonEquals = Not rightToken.IsKind(SyntaxKind.ColonEqualsToken)
```

Comment:
Same suggestion: we can combine this with `If(shouldAddColonEquals, s_colonEquals, Nothing)` here